libcore/AbstractSlot.vala

```diff
@@ -94,6 +94,7 @@ namespace GOF {
         public abstract void initialize_directory ();
         public abstract unowned GLib.List<GOF.File>? get_selected_files ();
         public abstract void set_active_state (bool set_active, bool animate = true);
```

Comment:
I understand this is out of the scope of the PR but this is redundant since already exists a signal to activate and signals can have a default handler.

Reply:
PRs to improve the codebase always welcome :-)

Reply:
Sure, I'll see what I can do.